core/server/admin.js

```diff
@@ -105,7 +105,10 @@ app.use("/Admin", express.static(path.join(__dirname, '../clients/Admin')));
 app.use("/_common", express.static(path.join(__dirname, '../clients/_common')));
 
 app.get('/', function(req, res){
-    res.sendFile(path.join(__dirname, '../clients/Admin/index.html'));
+    if(req.query.s === secret || DEBUG === true)
```

Comment:
* `PRIVATE.json` only contains a string?
* `DEBUG === true` => `DEBUG` (it's a boolean)